### ISSUE TITLE
the focus indicator is extremely ugly

### ISSUE THREAD

**Issue by Kommynct** [vector-im/element-web#25563]
### Your use case

I would like to either disable it, or make the focus indicator change based on the element that is in focus, instead of it being global the focus should be per-selector, for example, a better focus indicator for rooms would be highlighting the background (like when you hover), almost globally the hover indicator would look better

**Comment by t3chguy** [comment]
@gsouquet are Compound elements having focus indicators designed for them?

**Comment by gsouquet** [comment]
Yes. We have focus indicators for all elements in Compound.

I appreciate that the current design might not be to your taste but it is very unlikely we will provide an option to disable focus highlighting.

> make the focus indicator change based on the element that is in focus

Could you elaborate on the above, I believe this is already the case as elements have an orange outline to hint their focused state.

> a better focus indicator for rooms would be highlighting the background

Unfortunately only changing the background will not meet accessibility requirements https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

**Comment by Kommynct** [comment]
Ahh, didn't think of accessibility requirements, I don't want them disabled, I just thought they could be better integrated into the UI, but if it'll cause accessibility issues, I understand, i'm going to give a few examples of looking wonky though:

![image](https://github.com/vector-im/element-web/assets/13646790/0b6ba878-02a5-4c64-838a-17a2e4ccc715)

Around your avatar, it should be a circle if the avatars are circular:
![image](https://github.com/vector-im/element-web/assets/13646790/d5270cad-f77a-4f80-9faf-64836969d14e)

and when there's an expand button it doesn't properly surround it: ![image](https://github.com/vector-im/element-web/assets/13646790/4ef59c51-8618-45ab-b911-c4a2a9301a9c)
